CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### enhancement
 - Create E2E resources Helm chart by @juanjjaramillo in [#377](https://github.com/newrelic/newrelic-infra-operator/pull/377)
+- Create E2E tests by @juanjjaramillo in [#378](https://github.com/newrelic/newrelic-infra-operator/pull/378)
```

Comment:
Can you move this to be under the ## Unreleased section please?

Reply:
Nice catch! Done